### ISSUE TITLE
Timetable Management / Time table Setup / Create time table setup

### ISSUE THREAD

**Issue by M8Fayaz** [FayazM8/Groot-Second-Phase#46]
Bug Description: In the interval after noon, the mandatory star is missing,  also we planned to remove the break fields from mandatory.
Bug Screenshot or video: 
![image](https://github.com/FayazM8/Barathi-Institute-of-management-Groot-/assets/120467524/a1884af9-a6b9-49f3-825d-1969c456cbec)

**Comment by Diyanesh-M8** [comment]
yes, this is something that I brought up as well. We need to think whether we need to have both interval and lunch timings as mandatory. Because there are schools which closes before lunch time itself. So, my call is that schools and institutions who really have break timings to fill let them do it, others may just can skip it.
